bin/settings.py.template

```diff
@@ -55,4 +55,6 @@ SLACK_ALERT_DATA_USER_THRESHOLD = 100*1024*1024*1024
 SLACK_ALERT_DATA_CHANNEL_THRESHOLD = 4 * SLACK_ALERT_DATA_USER_THRESHOLD
 
 # Slack message to be sent when hard threshold is crossed (the one specified in SLACK_ALERT_DATA_USER_THRESHOLD or SLACK_ALERT_DATA_CHANNEL_THRESHOLD)
-SLACK_HARD_THRESHOLD_MESSAGE = '{user} your last query scanned {data_scanned_gb} GB'
\ No newline at end of file
+SLACK_HARD_THRESHOLD_MESSAGE_1 = '{user} your last query scanned {data_scanned_gb} GB'
+SLACK_HARD_THRESHOLD_MESSAGE_2 = 'example_text_2'
+SLACK_HARD_THRESHOLD_MESSAGE_3 = 'example_text_3'
```

Comment:
🟡 
Add example message that will explain its usage.

Handle in `hard_threshold_notification` `SLACK_HARD_THRESHOLD_MESSAGE_2/3`  logic values to be None or empty strings to not force users to use this feature

Reply:
Resolved 👍